### ISSUE TITLE
GraphQLErrorFromExceptionHandler doesn`t work for AbortExecutionException

### ISSUE THREAD

**Issue by Thinkenterprise** [graphql-java-kickstart/graphql-spring-boot#612]
I use the DDOS Feature Query Depth and configured a query depth of 3  ``maxQueryDepth: 3``. If the query depth is larger than 3 an ``AbortExecutionException`` will be thrown. Thats ok!! But the processing of the ``AbortExecutionException`` leads to an incorrect output -> A JSON rendering problem. The problem seems to be in the following method:

```
class GraphQLErrorFromExceptionHandler extends DefaultGraphQLErrorHandler {

  @Override
  protected List<GraphQLError> filterGraphQLErrors(List<GraphQLError> errors) {
    return errors.stream().map(this::transform).flatMap(Collection::stream)
        .collect(Collectors.toList());
  }
```` 
After this method is called the debugging session stops on ``@SneakyThrows`` in ``GraphQLObjectMapper``. 

```
 @SneakyThrows
  public String serializeResultAsJson(ExecutionResult executionResult) {
    return getJacksonMapper()
        .writeValueAsString(createResultFromExecutionResult(executionResult));
  }
```
If I disable the use of ``GraphQLErrorFromExceptionHandler`` it works!! - For disableing I have to set the property ``exception-handlers-enabled: false`` and no use of ``@ExceptionHandler`` annotations in my resolvers.

**Comment by oliemansm** [comment]
@Thinkenterprise There's no stack trace or anything being logged in this case? Also not when you set the log level to DEBUG?

**Comment by Thinkenterprise** [comment]
[stacktrace-and-debug.txt](https://github.com/graphql-java-kickstart/graphql-spring-boot/files/6445888/stacktrace-and-debug.txt)

**Comment by oliemansm** [comment]
@Thinkenterprise I was wrecking my brain how on earth this stack trace could happen seeing the code. But it appears the cause of this bug is already fixed on `master` and will therefore be available in release 11.1.0 that will be released shortly.

See also commit: https://github.com/graphql-java-kickstart/graphql-spring-boot/commit/3ae07c5fc8e9ba5395714a89173f8858c5481a69.
